packages/augur-sdk/src/state/db/AbstractDB.ts

```diff
@@ -60,7 +65,7 @@ export abstract class AbstractDB {
       result[prevDoc._id] = prevDoc;
       return result;
     }, {} as DocumentIDToDoc);
-    return await this.bulkUpsertDocuments(previousDocs, documents);
+    return this.bulkUpsertDocuments(previousDocs, documents);
```

Comment:
why remove await?

Reply:
@nuevoalex it becomes Promise<Promise<boolean>> which is the same as Promise<boolean>.

My editor tslnt's those things and tells me to remove them. Also, I googled it one day and ppl said to remove them since it's extra, unnecessary logic.